test/csv/interface/test_delegation.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestCSVInterfaceRead < Test::Unit::TestCase
+  def test_stringio_missing_methods_delegation
+    csv = CSV.new("h1,h2")
+
+    assert_equal(nil, csv.binmode?)
+    assert_equal(nil, csv.flock(0))
```

Comment:
I think that `NotImplementedError` is suitable instead of returning `nil`.
Because an application assumes this object is locked by calling this method is dangerous. This method never acquire a lock.

Reply:
Thanks for the catch!